src/config/file.js

```diff
@@ -0,0 +1,38 @@
+// Copyright 2019 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// ----------------------------------------------------------------------------
+//
+// http://www.simpletoken.org/
+//
+// ----------------------------------------------------------------------------
+
+'use strict';
+
+const fs = require('fs');
+
+/**
+ * Reads a configuration from a file and returns the object.
+ * @param {string} filePath The path to the config file to read.
+ */
+const readConfig = (filePath) => {
+  const configFile = fs.readFileSync(filePath);
```

Comment:
Do we maybe instead just want to `require` the `filePath`? This would also allow for `.js` files (which can have comments) for config, similar to how webpack, babel etc. do it.

Reply:
I thought about it. ESLint complains when `require`ing a variable instead of a static string. I think this is easily extendable in the future to support JS if we want to. It's as simple as updating this file to read JS as well. What do you think?

Reply:
👍  Doesn't have to happen in this PR.